docs/packaging-operator.md

```diff
@@ -4,7 +4,7 @@ This repository makes use of the [Operator Framework](https://github.com/operato
 
 ### Packaging format
 
-Your Operator submission can be formatted either following the `packagemanifest` or the newer `bundle` format. The former allows to ship your entire Operator with all its versions in one single directory. The latter allows shipping individual releases in container images. Both are supported but mixing of formats within a single Operator is not allowed. You need to decide for one or the other for your entire Operator listing.
+Your Operator submission can be formatted following the `bundle` or `package` format. The `package` format is a legacy format which is kept for backwards compatibility only and then, it strongly recommended to use `bundle` format. The former allows to ship your entire Operator with all its versions in one single directory. The latter allows shipping individual releases in container images. 
```

Comment:
Any reason not to leave the name of the old format as `packagemanifest`? That's what we've consistently referred to it as, and there are other places in the documentation that still reference that name.

Reply:
that was a mistake, Pushing the fix.